sky/build/sky_app.gni

```diff
@@ -129,7 +131,41 @@ template("sky_app") {
         deps += invoker.deps
       }
     }
-  } else if(is_ios) {
+  } else if(is_mac && defined(invoker.info_plist)) {
+    import("//build/config/mac/rules.gni")
+
+    mac_app(target_name) {
+
+      app_name = target_name + "_app"
```

Comment:
I'd really like us to fix the apps to be SkyShell.app and Asteroids.app and not sky_shell_app.app and game_app.app.